frappe/database/postgres/setup_db.py

```diff
@@ -78,10 +78,7 @@ def import_db_from_sql(source_sql=None, verbose=False):
 def get_root_connection(root_login=None, root_password=None):
 	if not frappe.local.flags.root_connection:
 		if not root_login:
-			root_login = frappe.conf.get("root_login") or None
-
-		if not root_login:
-			root_login = input("Enter postgres super user: ")
```

Comment:
passing the `--root-login` flag explicitly on non-standard (i.e. where root isn't `postgres`) seems like a very good trade-off.